python/cog/types.py

```diff
@@ -263,6 +261,9 @@ def get_filename(url: str) -> str:
 
 
 class ConcatenateIterator(Iterator[Item]):
+    def __next__(self) -> Item:
+        raise NotImplementedError("This is a stub class for typing purposes only")
```

Comment:
Is this right?

Reply:
This isn't obviously correct, and not worth the heartache. Backing out and ignoring instead.